### ISSUE TITLE
Could not find a valid dependency tree configuration with version 1.3.x

### ISSUE THREAD

**Issue by MaikKlein** [ParticlePeter/ErupteD#3]
This is in my current dub.sdl

```
dependency "erupted" version="~>1.3.0"
```

> Could not find a valid dependency tree configuration

There is no problem if I change the line to

```
dependency "erupted" version="~>1.1.0"
```

I sometimes get

`Package xlib-d contains invalid dependency xcb-d (ci=7)` So I assume it might be the problem of `xlib-d`.

**Comment by MaikKlein** [comment]
I also can't build erupted locally 
`Root package erupted contains reference to invalid package xcb-d >=2.1.0+1.11.1 <2.2.0`

I think that is because xlib-d uses only version 1.1.1. I can build it if I remove the `2.1.0+`.

I also made an issue here https://github.com/rtbo/xlib-d/issues/1

**Comment by ParticlePeter** [comment]
I got the same on Windows, so I thought it works only on Posix correctly. I fear that this is a x-lib Problem, nothing erupted could do about it, sorry. As a Workaround, if you use glfw3 on any platform, you don't Need any of the Vulkan/erupted platforms, glfw is handling them for you.

**Comment by MaikKlein** [comment]
Could you try the "fix" that I wrote in https://github.com/ParticlePeter/ErupteD/issues/2? I have a fully working triangle in vulkan with sdl and I would like to create a separate repository for it.

**Comment by MaikKlein** [comment]
This works for me

        {
            "name"          :   "dub-platform-xcb",
            "versions"      :   [ "VK_USE_PLATFORM_XCB_KHR" ],
            "dependencies"  :   {
                "xcb-d"         : "~>1.11.1"
            },

        {
            "name"          :   "dub-platform-xcb-derelict-loader",
            "versions"      :   [ "VK_USE_PLATFORM_XCB_KHR", "ERUPTED_FROM_DERELICT" ],
            "dependencies"  :   {
                "xcb-d"         : "~>1.11.1",
                "derelict-util" : "~>2.0.4"
            },

So that xcb in xlib and erupted have the same dependencies. I think that is a more general bug in dub though.

**Comment by ParticlePeter** [comment]
Your fix doesn't seem right. As explained in xcb-d readme v1.11.1 is the version of xcb, while v2.1.0 is the corresponding version of xcb-d. The xcb-d v1.11.1 is older, I am afraid that it is outdated.
Additionally, you also posted the xcb-d issue in the xlib-d repository. I suggest you get in touch with the xcb-d guy and ask him why it is not working, I was just following his readme.

**Comment by ParticlePeter** [comment]
Sorry, I somehow removed your post here. anyways here was my reply:
Your fix doesn't look right to me, v1.11.1 seems to be an older version and maybe outdated. I find Vilds versioning also kind of funky, however, if it isn't working you need to inform him.
You also posted the xcb-d issue in the xlib-d repository. I suggest you get in touch with the xcb-d guy and ask him why it is not working, I was just following his instructions.
Anyways, if you use xlib-d which references xcb-d v1.11.1 everything should still work smooth. You then would have both xcb-d versions in your dub package folder.
